interop-ribs/src/main/java/com/bumble/appyx/interop/ribs/InteropView.kt

```diff
@@ -46,9 +46,13 @@ internal class InteropViewImpl private constructor(
     }
 
     private fun retrieveIntegrationPoint(): IntegrationPoint {
-        val activity = context.findActivity<AppCompatActivity>()
+        val activity = context.findActivity<Activity>()
+        check(activity != null) {
+            "Could not find an activity from the context: $context"
+        }
         check(activity is IntegrationPointAppyxProvider) {
-            "Activity where InteropNode is used must implement IntegrationPointAppyxProvider"
+            "Activity where InteropNode is used must implement IntegrationPointAppyxProvider. " +
+                    "Activity: '${activity::class.qualifiedName}', Node: '${appyxNode::class.java}'"
```

Comment:
Why Kotlin reflection for activity and Java reflection for node?

Reply:
@CherryPerry whoops. mistype. sorry! fixed